src/imp/libc/process/auxv.rs

```diff
@@ -52,3 +52,15 @@ pub(crate) fn linux_execfn() -> &'static ZStr {
         zstr!("")
     }
 }
+
+/// Initialize process-wide state.
+#[cfg(any(
+    target_vendor = "mustang",
```

Comment:
What is `mustang`?

Reply:
Rust programs written entirely in Rust, with no libc: https://github.com/sunfishcode/mustang